src/pyfmi/fmi_algorithm_drivers.py

```diff
@@ -100,10 +100,23 @@ class AssimuloFMIAlgOptions(OptionBase):
             solver.
             Default: "Default"
 
+        dynamic_diagnostics --
+            If True, will enable logging of diagnostics data and requires that
+            the option 'result_handling' is set to 'binary', otherwise an exception is raised.
+            If True, the debug file will not be generated.
+            Before simulation start, a check for name clashes between

```

Comment:
State that it is "Diagnostics" that is forbidden, maybe something in line with:

Model variables are not allowed to start with "Diagnostics* using this option. This will be checked at simulation start and an exception raised if they do.